libraries/postgres/src/Database.js

```diff
@@ -396,7 +397,11 @@ class Database {
           await client.query(`DO ${dollarQuote(migrationScript)}`);
         }
         showProgress('..defining methods');
-        for (let [methodName, { args, body, returns}] of Object.entries(version.methods)) {
+        for (let [methodName, {deprecated, args, body, returns}] of Object.entries(version.methods)) {
+          // We don't need to do anything for deprecated methods
```

Comment:
I think in most cases we would want to update deprecated methods -- the method should be deprecated, but old code will still call it so it will also need an updated implementation that continues to function when the DB is upgraded.

In the "two step" process of upgrading from entities tables, it happens we don't want to do this -- the firs step updates the implementation, and the second step deprecates the old.  But in general I don't think this will be the case.